### PR TITLE
Fixes logo link on documentation site

### PR DESCRIPTION
**Pull request by goodlux**
This changes the link on the logo image of the PyTorch API documentation site to jump back to pytorch.org. 

Issue described here, (item number 2), on pytorch.github.io repo

https://github.com/pytorch/pytorch.github.io/issues/17

### CODE REVIEW

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
   <a style="color:#F05732" href="{{ theme_canonical_url }}{{ pagename }}.html">
     You are viewing unstable developer preview docs.
     Click here to view docs for latest stable release.
   </a>
 </div>
+
 {{ super() }}
 {% endblock %}
 
+{% block sidebartitle %}
```
Comment (SsnL) [inline docs/source/_templates/layout.html]: This definitely deserves comment that it is mostly copied from https://github.com/rtfd/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/layout.html#L92-L122 with modifications only to the url.

File: docs/source/_templates/layout.html [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+
+{% if logo and theme_logo_only %}
+  <a href="http://pytorch.org">
```
Comment (SsnL) [inline docs/source/_templates/layout.html]: https please

Instead of hardcoding, please add a variable to `docs/source/conf.py` and reference it here.

**Review by ssnl — CHANGES_REQUESTED** [review]

**Comment by weiyangfb** [comment]
@goodlux update on this?

**Comment by weiyangfb** [comment]
@goodlux are you still on this?

File: docs/source/_templates/layout.html (hunk 1 above)
Comment (goodlux) [reply docs/source/_templates/layout.html]: This whole layout was copied from that theme, originally. Much of it was removed, to simplify.

**Comment by soumith** [comment]
This will be conflicting with the changes that ShiftLab will be sending upstream (new theme / css and layout changes for docs), so I'm closing this PR. Thanks @goodlux

**Pull request closed without merging**